sleap_roots/bases.py

```diff
@@ -24,15 +25,17 @@ def get_bases(pts: np.ndarray, monocots: bool = False) -> np.ndarray:
 
 
 def get_base_tip_dist(
```

Comment:
the input traits always have to match the trait definition. This function is too complicated. We should simplify it to just assume taking in the expected input traits from from the trait graph.

Reply:
The `base_pts` and `tip_pts` can be used for our pipeline to quickly calculate the base-tip distance. The optional `pts` is designed for general purpose.